### PR TITLE
Core/Actions: Allow mounts buttons to be saved

### PR DESCRIPTION
**Pull request by Regigicas**
Currently you get Unknown action type 96 in console when putting a mount button in keybinds.

### CODE REVIEW

File: src/server/game/DataStores/DB2Stores.cpp [hunk 1]
```diff
@@ -488,6 +488,12 @@ MountEntry const* DB2Manager::GetMount(uint32 spellId) const
     return nullptr;
 }
 
+MountEntry const* DB2Manager::GetMountById(uint32 id) const
+{
+    auto mount = sMountStore.LookupEntry(id);
+    return mount ? mount : nullptr;
```
Comment (jackpoz) [inline src/server/game/DataStores/DB2Stores.cpp]:
```
return sMountStore.LookupEntry(id);
```

**Comment by MitchesD** [comment]
We can’t automatically merge this pull request. 

You have to resolve merge conflicts, also please squash commits ;-)

**Comment by Regigicas** [comment]
Done :)

File: src/server/game/Entities/Player/Player.cpp [hunk 2]
```diff
@@ -6179,6 +6179,22 @@ bool Player::IsActionButtonDataValid(uint8 button, uint32 action, uint8 type)
...
+                TC_LOG_ERROR("entities.player", "Mount action %u not added into button %u for player %s (%s): mount does not exist", action, button, GetName().c_str(), GetGUID().ToString().c_str());
+                return false;
+            }
+
+            if (!HasSpell(mount->SpellId))
+            {
+                TC_LOG_ERROR("entities.player", "Mount action %u not added into button %u for player %s (%s): Player does not know this mount", action, button, GetName().c_str(), GetGUID().ToString().c_str());
+                return false;
+            }
+            break;
```
Comment (RAlexis) [inline src/server/game/Entities/Player/Player.cpp]: Break after case braces.

Comment (MitchesD) [inline src/server/game/Entities/Player/Player.cpp]: not needed

Comment (RAlexis) [inline src/server/game/Entities/Player/Player.cpp]: Just saying because the other ones are like that.

Comment (Regigicas) [inline src/server/game/Entities/Player/Player.cpp]: The extra brace is needed because mount is initialized in the case otherwise it won't compile

Comment (MitchesD) [inline src/server/game/Entities/Player/Player.cpp]: but he means something different, don't touch it again, this is enough ;-)

**Pull request merged**